### ISSUE TITLE
Testing platform - force country does not work

### ISSUE THREAD

**Issue by katrine-olesen** [jackjonesfashion/wiz#379]
When using the testing tool, and when navigating between ROE markets. When having chosen a country, and then opening up the links in staging.
The preview of the staging does not follow the country you have chosen in the Testing platform tool. 

Eg. You have chosen to test IE. You open the links from the testing tool to check the categories. When staging opens up the links, the countries is different from the one you had just chosen in the testing platform. 

Let me know if i should show you or explain further

**Comment by jj-lucas** [comment]
- Open the tool, preview IT and date
- Click on slide
- Correct forcecountry, local and domain, but selected country is BE

**Comment by jj-lucas** [comment]
This seems like a bug in the session handling in DMW. Probably due to the cache and the continuous market changes under the same ".com" domain.
The market in the market selector will not update. The content in the page, however, will match the desired language.

e.g.
- Preview IT via slide
- Click on slide
- ROE slide, IT market in country selector
- Preview AT via tool
- Click on slide
- AT slide, IT market in country selector

Long story short, ignore the market being shown in the country selector, I'm pretty sure the correct market is loaded.

If you ever were to experience an unexpected slide / setup, let me investigate this further. Until then, consider this not an issue.
